src/tests/Common/CLRTest.Jit.targets

```diff
@@ -365,6 +372,139 @@ set DOTNET_JitPath=%CORE_ROOT%\superpmi-shim-collector.dll
     </PropertyGroup>
   </Target>
 
+  <!--
+  ***********************************************************************************************
+  IlasmRoundTrip for merged tests
+  ***********************************************************************************************
+  -->
+
+  <Target Name="IlasmRoundTripCommand" AfterTargets="AfterBuild" Condition="'$(IsMergedTestRunnerAssembly)' == 'true'">
+    <PropertyGroup>
+      <InputAssemblyName>$(AssemblyName).dll</InputAssemblyName>
+      <DisassemblyName>IL-RT/$(AssemblyName).il</DisassemblyName>
+      <TargetAssemblyName>IL-RT/$(AssemblyName).dll</TargetAssemblyName>
+      <_IlasmRoundTripScriptText><![CDATA[
+import os
+import shutil
+import subprocess
+import sys
+import glob
+import struct
+import platform
+
+def is_managed_assembly(file):
```

Comment:
This is useful functionality of course, but it should probably be a standalone utility. You can avoid having to deal with file format details if you use https://learn.microsoft.com/en-us/dotnet/standard/assembly/identify.

Reply:
It should be, but we can deal with it later.

Reply:
I bit the bullet and just wrote a tool to do the checks instead of the python code.